gdx/src/com/badlogic/gdx/graphics/g3d/utils/FirstPersonCameraController.java

```diff
@@ -54,26 +57,13 @@ public boolean keyUp (int keycode) {
 		return true;
 	}
 
-	/** Sets the velocity in units per second for moving forward, backward and strafing left/right.

```

Comment:
there is no reason to introduce a breaking change here, please revert setter removal and keep related fields protected.

Reply:
done